tensorboard/webapp/BUILD

```diff
@@ -31,6 +56,7 @@ ng_module(
         "//tensorboard/webapp/feature_flag",
         "//tensorboard/webapp/header",
         "//tensorboard/webapp/plugins",
+        "//tensorboard/webapp/plugins/text_v2",
```

Comment:
please remove this `text_v2` dep from `:app`

Reply:
I'm sorry, I don't understand why that is correct. Can you please elaborate?

Reply:
Done